api/models/network/updateNetwork.js

```diff
@@ -0,0 +1,62 @@
+import { isEqual, difference, values, some } from 'lodash'
+import setupNetworkAttrs from './setupNetworkAttrs'
+
+export default function updateNetwork (userId, id, params) {
+  if (!userId) throw new Error('updateNetwork called with no userID')
+  if (!id) throw new Error('updateNetwork called with no ID')
+  return setupNetworkAttrs(userId, params).then(attrs =>
+    bookshelf.transaction(transacting =>
+      // NOTE: EnsureLoad not built to work with belongsToMany relations
```

Comment:
It should work. What happened when you tried?

In any case you don't really need it here since this code is only good for use on a single network at once. If it were being passed in a Network instance then it would be a different story.